xbmc/cores/RetroPlayer/cheevos/Cheevos.cpp

```diff
@@ -26,17 +30,23 @@ namespace
 {
 // API JSON Field names
 constexpr auto SUCCESS = "Success";
-constexpr auto GAME_ID = "GameID";
 constexpr auto PATCH_DATA = "PatchData";
+constexpr auto GAME_ID = "GameID";
 constexpr auto RICH_PRESENCE = "RichPresencePatch";
+constexpr auto GAME_TITLE = "Title";
```

Comment:
would be nice to avoid the auto keyword here. What is the type of those vars? I assume they are cstrings / char*
There seems to be a recent "preference" for `std::string_view` instead